content/docs/user-guide/basic-concepts/dvc-project.md

```diff
@@ -15,6 +15,7 @@ match:
 
 Initialized by running `dvc init` in the **workspace** (typically a Git
 repository). It will contain the
-[`.dvc/` directory](/doc/user-guide/dvc-files-and-directories) and
-[DVC-files](/doc/user-guide/dvc-file-format) created with commands such as
+[`.dvc/` directory](/doc/user-guide/dvc-files-and-directories), as well as
+[`dvc.yaml`](/doc/user-guide/dvc-file-format) and
+[`.dvc` files](/doc/user-guide/dvc-file-format) created with commands such as
```

Comment:
In this case, it's easier to read it like this:
```suggestion
[`.dvc`](/doc/user-guide/dvc-file-format) files created with commands such as
```
So that "files" refers to both dvc.yaml and to .dvc .

Reply:
I'll just commit this. Please `git pull`.